test/gtest/ucp/ucp_test.h

```diff
@@ -266,4 +266,22 @@ std::ostream& operator<<(std::ostream& os, const ucp_test_param& test_param);
     UCP_INSTANTIATE_TEST_CASE_TLS(_test_case, self,   "self") \
     UCP_INSTANTIATE_TEST_CASE_TLS(_test_case, tcp,    "tcp")
 
+
+/**
+ * Instantiate the parameterized test case for all transport combinations
+ * with GPU memory awareness
+ *
+ * @param _test_case  Test case class, derived from ucp_test.
+ */
+#define UCP_INSTANTIATE_TEST_CASE_GPU_AWARE(_test_case) \
+    UCP_INSTANTIATE_TEST_CASE_TLS(_test_case, dcx,        "dc_x,cuda_copy,rocm_copy") \
```

Comment:
maybe `#define UCP_TEST_GPU_NO_IPC_TLS "cuda_copy,rocm_copy,gdrcopy,rocm_gdr" and use below

Reply:
done